plugins/net_plugin/net_plugin.cpp

```diff
@@ -629,7 +655,7 @@ namespace eosio {
       std::shared_ptr<tcp::socket>              socket; // only accessed through strand after construction
 
       fc::message_buffer<1024*1024>    pending_message_buffer;
-      std::atomic<std::size_t>         outstanding_read_bytes{0}; // accessed only from strand threads
+      std::size_t                      outstanding_read_bytes{0}; // accessed only from strand threads
```

Comment:
I'm sure there is a good reason, but I'm not getting why `std::atomic` is not needed?

Reply:
It is only accessed from each individual connection strand.

Reply:
Oh, yes, thanks!